extensions/wikia/CuratedContent/CuratedContentController.class.php

```diff
@@ -400,12 +392,62 @@ function getJsonItem( $titleName, $ns, $pageId ) {
 	 * @return bool
 	 */
 	static function onCuratedContentSave() {
-		self::purgeMethod( 'getList' );
+		global $wgServer;
+
+		$content = F::app()->wg->WikiaCuratedContent;
+
+		( new SquidUpdate( array_unique( array_reduce(
+			$content,
+			function ( $urls, $item ) use ( $wgServer ) {
+				if ( $item[ 'title' ] !== '' && empty( $item[ 'featured' ] ) ) {
```

Comment:
`empty` would return `true` when `$item[ 'title' ]` is `false`, `null`, empty array etc.